### DISCUSSION TITLE
Remote Scene Viewport

### DISCUSSION THREAD

**Discussion by theLemon505** [godotengine/godot-proposals#5927] [3D]
What if godot had the ability to view the remote scene tree at runtime? Unity has this feature and it makes the process of understanding if your scene is working correctly FAR easier. I don't think unreal has this feature so that might give godot a little plus over unreal. I have a basic understanding of how to approach this but am unfamiliar with godot's source code

**Comment by Zireael07** [comment]
Been suggested LOADS of times. 99% certain there's an open proposal for that, AND there was a PR that was unfortunately never finished. It's a WAAAY bigger undertaking than you seem to think

**Comment by Calinou** [comment]
This is already being tracked in https://github.com/godotengine/godot-proposals/issues/1864.
